src/inc/quic_datapath.h

```diff
@@ -858,6 +858,43 @@ CxPlatUpdateRoute(
     _In_ CXPLAT_ROUTE* SrcRoute
     );
 
+//
+// Get the RSS Configuration of the interface.
+//
+#define CXPLAT_RSS_HASH_TYPE_IPV4        0x001
+#define CXPLAT_RSS_HASH_TYPE_TCP_IPV4    0x002
+#define CXPLAT_RSS_HASH_TYPE_UDP_IPV4    0x004
+#define CXPLAT_RSS_HASH_TYPE_IPV6        0x008
+#define CXPLAT_RSS_HASH_TYPE_TCP_IPV6    0x010
+#define CXPLAT_RSS_HASH_TYPE_UDP_IPV6    0x020
+#define CXPLAT_RSS_HASH_TYPE_IPV6_EX     0x040
+#define CXPLAT_RSS_HASH_TYPE_TCP_IPV6_EX 0x080
+#define CXPLAT_RSS_HASH_TYPE_UDP_IPV6_EX 0x100
```

Comment:
Why are _types_ set up as _flags_? Can you set all of them simultaneously? I'd assume they are mutually exclusive.

Reply:
And, should these be an enum?

Reply:
This is copied from XDP, which copied from [NDIS RSS Hashing Types](https://learn.microsoft.com/en-us/windows-hardware/drivers/network/rss-hashing-types). They don't seem to be mutually exclusive but instead indicate how to handle packets that don't meet all the requirements. e.g. If a packet comes in that is not UDP/TCP, and the NIC doesn't support IPV4 or IPV6 RSS Hashing, then the packet is indicated on core0 (no RSS).